honeybee_radiance/primitive.py

```diff
@@ -424,22 +424,22 @@ def duplicate(self):
     @staticmethod
     def filter_dict_input(input_dict):
         """Filter a dictionary of a Primitive to get modifier and dependency objects."""
-        try:  # ensure the putil module is imported, which imports all primitive modules
-            putil
+        try:
```

Comment:
A note here will be helpful to know why you try to access the module and then import it hasn't been imported already.

Reply:
Right. The original reason has changed since, before this PR, it was done because it would import all of the modifier classes and it wasn't clear that we always wanted that. Now that we have to import all of the modifier classes in order to correctly parse the user libraries, the reason why it's in a try/except is to avoid a circular reference. I'll add a note about it.